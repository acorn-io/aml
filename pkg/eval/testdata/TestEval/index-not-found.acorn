x: {
	y: {
	}
}

a: [{
	name: "foo"
}]

d: function {
		b: x.y[a[0].name]
		return: b
}

y: d()
