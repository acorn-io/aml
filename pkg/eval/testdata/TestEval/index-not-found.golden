`key not found "foo": index-not-found.acorn:11:6 (11:6<-15:5)`
